tests/add-delete-add.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { test, expect } from '@playwright/test';
+
+test('testing adding interventions, then deleting one, then adding another', async ({ page }) => {
+    await page.goto('/scheme.html?authority=Derby#16.84/52.906457/-1.504519');
+    // wait for the map to load and interventions panel to appear
+    await page.getByText('Edit attributes Click an intervention to fill out its attributes Edit geometry N').waitFor();
+    // wait for router snapper to load so we can use route tool
+    await page.getByText('Route tool loading...').waitFor({ 'state': 'hidden' });
+
+    await page.getByRole('region', { name: 'Map' }).waitFor();
+    await page.getByRole('button', { name: 'New route' }).click();
```

Comment:
Not necessary yet, but maybe as we have more of these, some simple helper functions would be nice, so we can express something like `createRoute();  await page.getByRole('button', { name: '1) Untitled route' }).click(); await page.getByRole('button', { name: 'Delete' }).click(); createRoute()`. And maybe something to wait for the map + route tool to be ready too

Reply:
Yep, that would definitely be nice, most of the code here is generated from a codegen session but definitely up for abstracting this into helper functions.